modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -191,13 +191,22 @@ def get_json(confirmation_number, form_id)
         json
       end
 
-      def prepare_params_for_central_mail
+      def prepare_params_for_central_mail_and_log_error(e)
         params['veteran_full_name'] ||= {
           'first' => params['full_name']['first'],
           'last' => params['full_name']['last']
         }
         params['veteran_id'] ||= { 'ssn' => params['ssn'] }
         params['veteran_mailing_address'] ||= { 'postal_code' => @current_user.address[:postal_code] || '00000' }
+        Rails.logger.info(
+          'Simple forms api - 21-0966 Benefits Claims Intent to File API error,' \
+          'reverting to filling a PDF and sending it to Central Mail',
+          {
+            error: e,
+            current_user_participant_id: @current_user.participant_id,
```

Comment:
Change this to a presence check. `participant_id` is PII. This should log `true` or `false.